app/models/user.rb

```diff
@@ -0,0 +1,24 @@
+class User < ApplicationRecord
+  has_many :events_created, foreign_key: :creator_id, class_name: 'Event', dependent: :destroy
+  has_many :event_attendances, foreign_key: :event_attendee_id, dependent: :destroy
```

Comment:
You don't need to specify the `foreign_key` here since you already have that on the other side of the association.